macs/nix-darwin.nix

```diff
@@ -46,9 +46,7 @@ in
   };
 
   nix.package = pkgs.nixVersions.nix_2_24.overrideAttrs (oldAttrs: {
-    patches = oldAttrs.patches or [ ] ++ [
-      ./disable-chroot.patch
-    ];
+    patches = oldAttrs.patches or [ ] ++ [ ./disable-chroot.patch ];
```

Comment:
Unrelated formatting change. This snuck in in https://github.com/NixOS/infra/commit/dfd77d6dbe22ebbc2d8720d217a9947c02741723